xlators/mgmt/glusterd/src/glusterd-utils.c

```diff
@@ -7706,10 +7706,14 @@ glusterd_is_path_in_use(char *path, gf_boolean_t *in_use, char **op_errstr)
 
     if (*in_use) {
         if (path && curdir && !strcmp(path, curdir)) {
-            snprintf(msg, sizeof(msg),
-                     "%s is already part of a "
-                     "volume",
-                     path);
+            int z = snprintf(msg, sizeof(msg),
```

Comment:
I think we don't need to declare a new variable , we can use ret instead of using z.

Reply:
Using ret is causing a problem which i found out in previous run regression when we are assigning value to ret of snprintf when it is successful it can also take value other than zero but then in caller function of glusterd_is_path_in_use() will treat ret value as an error and proceed according to that

Reply:
ok